Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1325 @@
+    ioc_query = argToList(args.get('ioc_query'))
+    severity = int(args.get('severity'))
+    timestamp = convert_date_to_unix(args.get('timestamp'))
+    ioc_contents = []
+    iocs = assign_params(
+        ipv4=ipv4,
+        ipv6=ipv6,
+        dns=dns,
+        md5=md5,
+        query=ioc_query
+    )
+    headers = ['ID', 'Title', 'Timestamp', 'Description', 'Severity', 'Link', 'IOCs_v2', 'Tags', 'Visibility']
+    result = client.create_report_request(title, description, tags, severity, iocs, timestamp)
+
+    contents = {
+        'ID': result.get('id'),
+        'Timestamp': convert_unix_to_timestamp(result.get('timestamp')),
+        'Description': result.get('description'),
+        'Title': result.get('title'),
+        'Severity': result.get('severity'),
+        'Tags': result.get('tags'),
+        'Link': result.get('link'),
+        'Visibility': result.get('visibility')
+    }
+
+    context = {
+        'ID': result.get('id'),
+        'Timestamp': convert_unix_to_timestamp(result.get('timestamp')),
+        'Description': result.get('description'),
+        'Title': result.get('title'),
+        'Severity': result.get('severity'),
+        'Tags': result.get('tags'),
+        'Link': result.get('link'),
+        'IOCs': result.get('iocs_v2'),
+        'Visibility': result.get('visibility')
+    }
+
+    iocs = result.get('iocs_v2', {})
+    for ioc in iocs:
+        ioc_contents.append({
+            'ID': ioc.get('id'),
+            'Match_type': ioc.get('match_type'),
+            'Values': ioc.get('values'),
+            'Field': ioc.get('field'),
+            'Link': ioc.get('link')
+        })
+
+    readable_output = tableToMarkdown(f'The report was created successfully.', contents, headers, removeNull=True)
+    ioc_output = tableToMarkdown(f'The IOCs for the report', ioc_contents, removeNull=True)
+    results = CommandResults(
+        outputs_prefix='CarbonBlackEEDR.Report',
+        outputs_key_field='ID',
+        outputs=context,
+        readable_output=readable_output + ioc_output,
+        raw_response=result
+    )
+    return results
+
+
+def ignore_report_command(client: Client, args: dict) -> str:
+    report_id = args.get('report_id')
+
+    client.ignore_report_request(report_id)
+
+    return f'The report with report_id "{report_id}" and all contained IOCs will not match future events ' \
+        f'for any watchlist.'
+
+
+def reactivate_report_command(client: Client, args: dict) -> str:
+    report_id = args.get('report_id')
+
+    client.reactivate_report_request(report_id)
+
+    return f'Report with report_id "{report_id}" and all contained IOCs will match future events for all watchlists'
+
+
+def get_report_ignore_status_command(client: Client, args: dict) -> str:
+    report_id = args.get('report_id')
+
+    result = client.get_report_ignore_status_request(report_id)
+
+    if not result.get('ignored'):
+        return f'ignore status for report with report_id "{report_id}" is false.'
```

Comment:
that is the terminology in the ui? report id status can be true or false?

Reply:
@Itay4 Here I couldn't find it in the UI, in the API it's true or false.